Training/Training/file.c

```diff
@@ -7,7 +7,51 @@
 // ------------------------------------------------------------------------------------------------
```

Comment:
You should not name programs as "file.c" or "program.c". Only school students or 1st year college students do this. Professionals give meaningful names.